docs/configuration/command_line_interface.rst

```diff
@@ -4,3 +4,173 @@ Command Line Interface (CLI) commands
+===============
+These are the commands you may need to use in relation to your Mautic instance. Add a ``bin/console`` before Mautic command.
+
+**Example**
+
+.. code-block:: shell
+
+  bin/console mautic:segments:update
+
+.. vale off
+
+.. list-table:: 
+   :widths: 100 100
+   :header-rows: 1
+
+   * - Command
+     - Description
+   * - mautic:assets:generate
+     - Combines and minifies Asset files from each bundle into single production files
+   * - mautic:broadcasts:send
+     - Process Contacts pending to receive a Channel broadcast.
+   * - mautic:campaigns:execute
+     - Execute specific scheduled events.
+   * - mautic:campaigns:messagequeue
+     - Process sending of messages queue.
+   * - mautic:campaigns:messages
+     - Process sending of messages queue.
+   * - mautic:campaigns:rebuild
+     - Rebuild Campaigns based on Contact Segments.
+   * - mautic:campaigns:trigger
+     - Trigger timed events for published Campaigns.
+   * - mautic:campaigns:update
+     - Rebuild Campaigns based on Contact Segments.
+   * - mautic:campaigns:validate
+     - Validate if a Contact has been inactive for a decision and execute events if so.
+   * - mautic:citrix:sync
+     - Synchronizes registrant information from Citrix products
+   * - mautic:cache:clear
+     - Clears the cache 
+   * - mautic:contacts:deduplicate
+     - Merge contacts based on same unique identifiers
+   * - mautic:custom-field:create-column
+     - Creates the actual column in the table
+   * - mautic:email:fetch	
+     - Fetch and process monitored Email.
+   * - mautic:emails:send
+     - Processes SwiftMail's mail queue
+   * - mautic:import
+     - Imports data to Mautic
+   * - mautic:integration:fetchleads
+     - Fetch Contacts from Integration.
+   * - mautic:integration:pipedrive:fetch
+     - Pulls the data from Pipedrive and sends it to Mautic
+   * - mautic:integration:pipedrive:push
+     - 	Pushes the data from Mautic to Pipedrive
+   * - mautic:integration:pushactivity
+     - Push Contact activity to Integration.
+   * - mautic:integration:pushleadactivity
+     - Push Contact activity to Integration. 
+   * - mautic:integration:synccontacts
+     - Fetch Contacts from Integration.
+   * - mautic:install:data
+     - Installs data
+   * - mautic:iplookup:download
+     - Fetch remote datastores for IP lookup services that leverage local lookups.
+   * - mautic:maintenance:cleanup
+     - Cleans up older data.
+   * - mautic:messages:send
+     - Process sending of messages queue.
+   * - mautic:migrations:generate
+     - Generate a blank migration class.
+   * - mautic:plugins:install
+     - Installs Plugins.
+   * - mautic:plugins:reload
+     - Reloads Plugins.
+   * - mautic:plugins:update
+     - Updates Plugins.
+   * - mautic:queue:process
+     - Process queues
+   * - mautic:reports:scheduler
+     - Processes scheduler for Report's export
+   * - mautic:segments:check-builders
+     - Compare output of query builders for given Segments
```

Comment:
Will be removed in M5 (https://github.com/mautic/mautic/pull/11041) as it does nothing at the moment.